exonum/benches/criterion/block.rs

```diff
@@ -142,29 +142,43 @@ mod timestamping {
         }
     }
 
-    transactions! {
-        TimestampingTransactions {
-            struct Tx {
-                data: &Hash,
-            }
+    #[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert)]
+    #[protobuf_convert("tests::BenchTsTx")]
+    #[exonum_derive_outer]
```

Comment:
I agree this is bad name, but there is only one place where this solution is needed(benchmarks and external tests in exonum crate).
Yes attribute map will look better.

Reply:
I fixed it, and named it exonum_root_path = "crate/self". It will be removed after in future rust you could reexport exonum crate from itself under exonum namespace.